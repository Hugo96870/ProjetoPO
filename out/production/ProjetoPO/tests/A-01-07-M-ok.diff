*** tests/A-01-07-M-ok.out	2020-11-10 08:52:15.000000000 +0000
--- tests/A-01-07-M-ok.outhyp	2020-12-03 15:04:27.787430100 +0000
***************
*** 94,100 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 94,100 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
***************
*** 106,109 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 106,109 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
