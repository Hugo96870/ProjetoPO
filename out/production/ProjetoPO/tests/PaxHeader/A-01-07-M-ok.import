30 mtime=1604998335.172956493
