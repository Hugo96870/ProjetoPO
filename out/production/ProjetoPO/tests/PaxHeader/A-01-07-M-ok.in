30 mtime=1604998335.173544858
