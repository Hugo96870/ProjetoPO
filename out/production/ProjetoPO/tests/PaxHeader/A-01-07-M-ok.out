30 mtime=1604998335.174261532
